packages/ma-components/local/page-footer/page-footer.tsx

```diff
@@ -0,0 +1,43 @@
+import { PropsWithChildren, type ReactElement } from 'react';
+import './page-footer.css';
+
+export interface PageFooterProps {
+  sections: ReactElement[];
+  metaLinks: ReactElement[];
+}
+
+export function PageFooter(props: PropsWithChildren<PageFooterProps>) {
+  return (
+    <footer className="ma-page-footer">
+      <div className="ma-page-footer__content">{props.sections}</div>
+
+      <nav className="ma-page-footer__meta">
+        <ul className="ma-page-footer__meta-content">
+          {props.metaLinks.map((child, index) => (
+            <li key={index}>{child}</li>
+          ))}
+        </ul>
+      </nav>
+    </footer>
+  );
+}
+
+export interface PageFooterSectionProps {
+  linkUrl?: string;
+  linkLabel?: ReactElement | string;
+  linkIcon?: ReactElement;
+}
+
+export function PageFooterSection(props: PropsWithChildren<PageFooterSectionProps>) {
+  return (
+    <section className="ma-page-footer__section">
+      <div className="ma-page-footer__section-content">{props.children}</div>
+      {props.linkUrl && props.linkLabel && (
+        <a className="utrecht-button utrecht-button--primary-action" href={props.linkUrl}>
```

Comment:
Als je `<Button label={props.linkLabel}>` doet, dan komt 'ie in een spannetje te staan waardoor je zonder risico op teveel gaps de column-gap design token kan gebruiken voor afstand tussen label en icon.

Hopelijk binnenkort ook een optie voor de Utrecht `<Link>` trouwens: https://github.com/nl-design-system/utrecht/pull/2797

Reply:
ik snap niet helemaal wat je bedoeld. Bedoel je de `<a>` op regel 36 te vervangen door de `<Button />`? Dat wordt toch geen `<a>` element? En de `<Link />` heeft de functie nog niet.

Reply:
Oh, die a had ik niet gespot. Dan moet je de LinkButton hebben of de ButtonLink - never can remember which one 😄

Reply:
Bedoel je deze? https://github.com/nl-design-system/utrecht/blob/main/packages/component-library-react/src/ButtonLink.tsx hier wordt geen span gerenderd toch? en de `LinkButton` rendert een daadwerkelijke `<button>`

Reply:
Ik heb uiteindelijk de ButtonLink gebruikt (niet hier maar als value van de `action` prop).